README.md

```diff
@@ -127,4 +127,31 @@ Follow the instructions to login to GitHub using your credentials.
 Follow the instructions specified, then run this command again:<br/>
 `git push origin {your-branch-name}`
 
+## Testing of Sample Queries
+Once you've added/updated the sample queries, you can test them out by going to [Graph Explorer](https://developer.microsoft.com/en-us/graph/graph-explorer), and appending this to the Graph Explorer url.
+
+`?devx-api=https://graphexplorerapi.azurewebsites.net&org=microsoftgraph&branchName=dev`
+
+
+This overrides the default DevX API url that fetches from blob storage and instead, fetches the files from your branch by adding two optional query params to the url i.e **org** and **branchName** to represent the path of the file.
```

Comment:
I think we can abstract out the technical details and just highlight that this would fetch the permissions/samples from the specified repo or branch.